mautic/mautic
# Focus items in dynamic web content

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for features or enhancements / 3.2 for bug fixes 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 


#### Description:
Issue: When using the

{focus=_ID#_}

token in dynamic web content, the js link is relative rather than absolute. For example, instead of src="http://your.etsu.edu/focus/20.js", it's src=”/focus/20.js”.

#### Steps to reproduce the bug:
1. Use test steps. Focus Item does not work

#### Steps to test this PR:
1. Create focus item and publish it
2. Create DWC 
 	- Use content `MAUT-4230 DWC content {focus=1}` where `1` is id of your created focus item
	- Is campaign based = no
	- Requested Slot Name = `slot1`
	- Add filter - `Country = Czech Republic`
3. Create landing page 
	- with dynamic content where
	- Slot name is `slot1`
	- Slot content is `Dynamic web content applied {dwc=slot1}‍ `
4. Use incognito mode in browser
	- add conrtact via form in incognito mode - contact should be added to CS
	- try to show page created in step 3 in incognito mode - focus item is not displayed
	- in CS set country to Czech Republic for this contact
	- reload page in incognito mode - focus item is displayed - JS is loaded

![Snímek obrazovky 2020-12-11 v 13 43 28](https://user-images.githubusercontent.com/12815758/101906056-a6990700-3bb8-11eb-8a0c-a7150ef53233.png)
![Snímek obrazovky 2020-12-11 v 13 46 24](https://user-images.githubusercontent.com/12815758/101906063-a993f780-3bb8-11eb-99c7-65682be2a7f1.png)
![Snímek obrazovky 2020-12-11 v 13 46 35](https://user-images.githubusercontent.com/12815758/101906069-abf65180-3bb8-11eb-9451-836af54dc30b.png)
